content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -9,61 +9,71 @@ weight: 20
 
-Some resource types will have one or more sub-resources, represented as sub paths below the resource:
+Some resource types have one or more sub-resources, represented as sub paths below the resource:
 
-* Cluster-scoped subresource: `GET /apis/GROUP/VERSION/RESOURCETYPE/NAME/SUBRESOURCE`
-* Namespace-scoped subresource: `GET /apis/GROUP/VERSION/namespaces/NAMESPACE/RESOURCETYPE/NAME/SUBRESOURCE`
+* Cluster-scoped subresource: `GET /apis/<GROUP>/<VERSION>/<RESOURCETYPE>/<NAME>/<SUBRESOURCE>`
+* Namespace-scoped subresource: `GET /apis/<GROUP>/<VERSION>/namespaces/<NAMESPACE>/<RESOURCETYPE>/<NAME>/<SUBRESOURCE>`
 
 The verbs supported for each subresource will differ depending on the object - see the API documentation more information. It is not possible to access sub-resources across multiple resources - generally a new virtual resource type would be used if that becomes necessary.
 
 
 ## Efficient detection of changes
 
-To enable clients to build a model of the current state of a cluster, all Kubernetes object resource types are required to support consistent lists and an incremental change notification feed called a **watch**.  Every Kubernetes object has a `resourceVersion` field representing the version of that resource as stored in the underlying database. When retrieving a collection of resources (either namespace or cluster scoped), the response from the server will contain a `resourceVersion` value that can be used to initiate a watch against the server. The server will return all changes (creates, deletes, and updates) that occur after the supplied `resourceVersion`. This allows a client to fetch the current state and then watch for changes without missing any updates. If the client watch is disconnected they can restart a new watch from the last returned `resourceVersion`, or perform a new collection request and begin again. See [Resource Version Semantics](#resource-versions) for more detail.
+To enable clients to build a model of the current state of a cluster, all Kubernetes object resource types are required to support consistent lists and an incremental change notification feed called a **watch**.  Every Kubernetes object has a `resourceVersion` field representing the version of that resource as stored in the underlying database. When retrieving a collection of resources (either namespace or cluster scoped), the response from the server will contain a `resourceVersion` value that can be used to initiate a Watch against the server. The server will return all changes (creates, deletes, and updates) that occur after the supplied `resourceVersion`. This allows a client to fetch the current state and then watch for changes without missing any updates. If the client Watch is disconnected they can restart a new Watch from the last returned `resourceVersion`, or perform a new collection request and begin again. See [Resource Versions](#resource-versions) for more detail.
```

Comment:
Could you use code block formatting around the `watch` term instead of double asterisks or capitalization?
Can you cleanup the "will contain" to "contains" and "will return" to "returns"
Edit: 
If a client is disconnected from a `watch`, they can restart a new `watch` from the last returned `resourceVersion`, or perform a new collection request to receive updates again. See [Resource Versions](#resource-versions) for more detail.

Reply:
I've just realized that there are three meanings of “watch” in Kubernetes - it's a verb, `watch` (eg in RBAC); it's a noun, `Watch`, as well (representing a structure in an API response for example), and then the English word watch describes what you use those jargon terms to achieve.

I guess I need to work out which I think each of these is closest to and adjust styling to match.

Reply:
Now addressed, I think.